monkestation/code/modules/ghost_players/job_helpers/firing_range_helper.dm

```diff
@@ -53,6 +53,7 @@
 		/obj/item/gun/magic/wand/safety,
 		/obj/item/gun/medbeam,
 		/obj/item/gun/energy/recharge/kinetic_accelerator/meme,
+		/obj/item/gun/magic/artifact,
```

Comment:
perhaps add a gondola meat blacklist to the food one aswell considering this is the anti-escape pr

Reply:
agreed. the food dispenser should disallow kudzu (if not already) and disallow gondola meat.